doc/tutorial/fcm_fundamentals

```diff
@@ -0,0 +1,859 @@
+                The main type of files that we will be committing to a repository are: 
+                ASCII and binary files.
+                <br/>
+                <ul>
+                    <li>Assci files can be incrementally updated where only the changed or new 
+                        part is added to the file, for example a txt file, code, LaTeX file.
+                    </li>
+                    <br/>
+                    <li>Binary files are generally larger than ASCII and are only "visible" as a whole 
+                        file and therefore cannot be incrementally updated which means it is often a 
+                        large file which with each new version is saved as a new file and therefore 
+                        the repository can grow very very fast, example files of these files are word 
+                        documents.
+                        These can be saved but it is best to save them once they are complete and 
+                        finalised.
+                        We may wish to write documentation in a text editor or LaTeX to maintain a 
+                        version controlled document.
+                    </li>
+                </ul>
+            </div>
+            <div class="page-header">
+                <h3>Diagrams to explain ideas and concepts</h3>
+            </div>
+            <div class="well">
+                <pre>
+&lt;root&gt;
+    |
+    |-- &lt;project 1&gt;
+    |       |
+    |       |-- trunk
+    |       |-- branches
+    |
+    |-- &lt;project 2&gt;
+    |       |
+    |       |-- trunk
+    |       |-- branches
+    |
+    |-- ...
+
+</pre>
+                The structure of a repository.
+                <br/>
+                <br/>
+                <pre>
+                &lt;Trunk&gt;
+                   |
+    &lt;Branch3&gt;------|---------&lt;Branch2&gt;------&lt;Branch1&gt;
+       |           |#1          |              |
+       |           |            |#2            |
+       |#3         |            |              |
+       |           |#4          |              |
+       |           |            |              |#5
+       |           |            |              |#6
+       |           |            |              |
+       |           |#7          |              |
+       |#8         |            |              |
+       |           |            |#9            |
+       |-----------|#10         |              |
+                   |#11---------|              |
+                   |                           |#12
+                   |#13------------------------|
+                   |
+                   |#14
+
+</pre>
+                The concept of revision number. Every change that occurs results in an increase
+                in the revision number, shown here by the <samp>#</samp>.
+                <br/>
+                <br/>
+                <img src="merging.png" id="I1">
+                <br/>
+                Best working practice for merging.
```

Comment:
Could you maybe split the image in half here. Explain what's going on in the first image. Then put the second image in, explain what's going on and why its bad/non-optimal.